docs/source/mechanical_analyzers/SPM_rotor_speed_limit_analyzer.rst

```diff
@@ -0,0 +1,224 @@
+
+.. code-block:: python
+
+   ######################################################
+   #Setting the machine geometry and operating conditions
+   ######################################################
+   r_sh = 5E-3 # [m]
+   d_m = 2E-3 # [m]
+   r_ro = 12.5E-3 # [m]
+   deltaT = 0 # [K]
+   N_max = 100E3 # [RPM]
+   d_sl=1E-3 # [m]
+   delta_sl=-2.4E-5 # [m]
+
+   ######################################################
+   #Creating problem
+   ######################################################
+   problem = rsl.SPM_RotorSpeedLimitProblem(r_sh, d_m, r_ro, d_sl, delta_sl, deltaT, 
+                                        N_max, mat_dict, mat_failure_dict)
+
+
+Example with No Rotor Sleeve
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+To analyze a rotor with no sleeve, a simple set of ``d_sl``, ``delta_sl``, and ``deltaT`` are required when creating the problem. This is shown in the following code:
+
+.. code-block:: python
+
+   ######################################################
+   #Setting the machine geometry and operating conditions
+   ######################################################
+   r_sh = 5E-3 # [m]
+   d_m = 2E-3 # [m]
+   r_ro = 12.5E-3 # [m]
+   deltaT = 0 # [K]
+   N_max = 100E3 # [RPM]
+   d_sl=0 # [m]
+   delta_sl=0 # [m]
+
+   ######################################################
+   #Creating problem
+   ######################################################
+   problem = rsl.SPM_RotorSpeedLimitProblem(r_sh, d_m, r_ro, d_sl, delta_sl, deltaT, 
+                                        N_max, mat_dict, mat_failure_dict)
+
+
+
+Outputs to User
+***********************************
+To initialize an instance of the analyzer class ``SPM_RotorSpeedLimitAnalyzer``, the user must specify the RPM evaluation step size ``N_step`` in units of *RPM* and 
+number of rotor nodes ``node`` (for evaluating rotor stress) when defining the analyzer object. Once the analyzer class has been defined, the user can call the 
+``.analyze`` method and input the defined instance of ``SPM_RotorSpeedLimitProblem`` problem class. The script will run through the code at an incremental speed 
+increases (``N_step`` defined by the user) to determine the failure speed and material.
+
+Since this analyzer only provides an estimate of RPM failure speed, the user should consider user a coarse `N_step` value (such as 1000 RPM) to speed up the analysis. 
+For the `node` value, the user can also adjust accordingly based on their machine rotor size. In addition, the user should consider implementating a factor of safety 
+for the machine speed limit in their design.
+
+Use the following code to run the aforementioned example analysis:
+
+.. code-block:: python
+
+   ######################################################
+   #Creating analyzer class
+   ######################################################
+   analyzer = rsl.SPM_RotorSpeedLimitAnalyzer(N_step=100,node=1000)
+   result = analyzer.analyze(problem)
+   print(result.failure_mat)
+   print(result.speed)
```

Comment:
Everything above line 189 (including this example) belongs in the `Input from User` section

Reply:
This was changed in latest commit